templates/.reek.yml

```diff
@@ -0,0 +1,40 @@
+---
```

Comment:
no option has been changed. they just got nested under the scopes `detectors` and `directories`